layouts/page/superpowers.html

```diff
@@ -0,0 +1,276 @@
+                    </div>
+                    <div class="pl-6 pr-12">
+                        <h3 class="font-normal text-gray-800 text-3xl mt-0">
+                            <a href="{{ relref . "docs/guides/crossguard" }}">
+                                <span class="align-middle text-blue-700 transition-all hover:mr-1">Policy</span>
+                                <span class="text-blue-400 text-lg">&rarr;</span>
+                            </a>
+                        </h3>
+                        <p class="text-lg text-gray-700 mb-12 pb-12 border-b border-gray-1 md:border-b-0 leading-relaxed">
+                            Enforce standards for deployments with auditing capabilities across roles,
+                            environments, projects or organizations. Define security, compliance, and
+                            cost controls to prevent mistakes from reaching production.
+                        </p>
+                    </div>
+                </li>
+                <li class="md:w-1/2 flex items-start">
+                    <div class="bg-blue-100 border-4 border-blue-200 rounded-full p-6" style="fill: #4387c7;">
+                        <svg class="h-12 w-12" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512"><defs><style>.fa-secondary{opacity:.4}</style></defs><path d="M161.6 288H96a96 96 0 0 1 0-192c.6 0 1.1.2 1.6.2C105.3 41.9 151.6 0 208 0a111.61 111.61 0 0 1 104.5 72.7A95.07 95.07 0 0 1 352 64a96 96 0 0 1 96 96 93 93 0 0 1-7 34.7 110.5 110.5 0 0 0-31.6 11.8A142.54 142.54 0 0 0 304 160c-73.9 0-134.3 56.2-142.4 128z" class="fa-secondary"/><path d="M640 416a96 96 0 0 1-96 96H224a96 96 0 0 1-96-96c0-42.5 27.8-78.2 66.1-90.8A113.72 113.72 0 0 1 192 304a111.94 111.94 0 0 1 112-112c43.2 0 80.4 24.9 99 60.8 14.7-17.5 36.4-28.8 61-28.8a80 80 0 0 1 80 80 78.09 78.09 0 0 1-1.6 16.2c.5 0 1-.2 1.6-.2a96 96 0 0 1 96 96z" class="fa-primary"/></svg>
+                    </div>
+                    <div class="pl-6 pr-12">
+                        <h3 class="font-normal text-gray-800 text-3xl mt-0">
+                            <a href="{{ relref . "/docs/get-started" }}">
+                                <span class="align-middle text-blue-700 transition-all hover:mr-1">Any cloud, any language</span>
```

Comment:
This feels a bit odd to be a superpower.

Reply:
Two thoughts here:

1. We made a big deal in the roadmap about multi-language support and language talks feature prominently, but, we kind of leave this out of 'superpowers'
2. I actually added this to balance out the 2x3 list on this page.  

Happy to keep or drop if this feels artificial.